docs/community/contribute/nearcore.md

```diff
@@ -149,14 +150,14 @@ lto=off`.
 
 Requires installing lld linker.
 
-```bash
-# cargo clean
-# time RUSTFLAGS='-C lto=off -C link-arg=-fuse-ld=lld' RUSTC_WRAPPER= cargo build -p neard --release
+```console
+$ cargo clean
+$ time RUSTFLAGS='-C lto=off -C link-arg=-fuse-ld=lld' RUSTC_WRAPPER= cargo build -p neard --release
```

Comment:
NB: this is going to be shortly made more straightforward via [`-Clinker-flavor=gcc:lld`](https://github.com/rust-lang/rust/pull/96827), with or without `-Clink-self-contained=linker`. We should keep an action item for that.